modules/cms-kit/src/Volo.CmsKit.Admin.HttpApi/Volo/CmsKit/Admin/Contents/ContentController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+using Volo.Abp;
+using Volo.Abp.Application.Dtos;
+using Volo.CmsKit.Admin.Contents;
+using Volo.CmsKit.Permissions;
+
+namespace Volo.CmsKit.Admin.Contents
+{
+    [Authorize(CmsKitAdminPermissions.Contents.Default)]
+    [RemoteService(Name = CmsKitCommonRemoteServiceConsts.RemoteServiceName)]
+    [Area("cms-kit")]
+    [Route("api/cms-kit-admin/contents")]
+    public class ContentController : CmsKitAdminController, IContentAdminAppService
```

Comment:
Let's rename this to the `ContentAdminController`. We need to use the same convention.